source/css/_schemes/Gemini/index.styl

```diff
@@ -78,7 +78,7 @@ use_seo = hexo-config('seo');
 // Post delimiters.
 .post-eof,
 .post-spread {
-  hide();
+  display: none !important;
```

Comment:
Need to recheck here, because values in base without `!imortant`.

https://github.com/theme-next/hexo-theme-next/blob/9876b88e8cd2925901e3743eb73c945909c7aa70/source/css/_mixins/base.styl#L71-L77

Reply:
Yes, I have tested, this is an exception. All other ‘display: none’ style can be replaced with macro ‘none()’

Reply:
So, pneed to fix it somehow.

Reply:
No better solution, I think.

Or we can remove `show`, `hide` and `circle` macros. Currently, they are only used a few times.

Reply:
So, need any solution. For now you suggest Improvment (Optimization), but give 1 additional Bug  with this.

Reply:
`display: none !important` is used to overwrite the `display: block` style in `.post-eof` in scheme Gemini. To make sure it works on all browsers, an `!important` is needed. It won't cause additional bug.

Reply:
Remove `show`, `hide` and `circle` macros is another way. What's your opinion?

Reply:
> `display: none !important` is used to overwrite the `display: block` style in `.post-eof` in scheme Gemini. To make sure it works on all browsers, an `!important` is needed. It won't cause additional bug.

I know it, why u say me what I already know? Question is: why there are used `!important`? What rewrited? Why only for Gemini? How to make without rewriting (without `!important`)? Is it possible? If possible, need to try it, yeah?

Reply:
Oh, I understand. I'll open another PR to reduce the usage of `!important`.